src/composables/useTxStatus.ts

```diff
@@ -1,14 +1,69 @@
-const pendingCount = ref(0);
+import { useStorage } from '@vueuse/core';
+import getProvider from '@snapshot-labs/snapshot.js/src/utils/provider';
+import { PendingTransaction } from '@/helpers/interfaces';
+
+const PENDING_TRANSACTIONS_STORAGE_KEY = 'snapshot.pendingTransactions';
+const pendingTransactions = useStorage(
```

Comment:
I don't think useStorage will work outside of the composable function

Reply:
https://github.com/vueuse/vueuse/issues/464

Reply:
Ah nice one!